game/play_game.go

```diff
@@ -0,0 +1,188 @@
+package game
+
+import (
+	"fmt"
+	"sync"
+)
+
+func (g *Game) Play() error {
+	for !g.IsOver() {
+		var err error
+		switch g.round.CurrentStage {
+		case Deal:
+			err = g.dealPhase()
+			g.round.CurrentStage = BuildCrib
+		case BuildCrib:
+			err = g.buildCrib()
+			g.round.CurrentStage = Cut
+		case Cut:
+			err = g.cutPhase()
+			g.round.CurrentStage = Pegging
+		case Pegging:
+			err = g.peg()
+			g.round.CurrentStage = Counting
+		case Counting:
+			fmt.Printf("Counting players hands\n")
```

Comment:
We should remove the prints in this file (and all non-error cases not in "terminal interaction")